interfaces/api/v1/audit/chapter_repair_routes.py

```
@@ -0,0 +1,122 @@
+    from interfaces.api.dependencies import get_chapter_repair_service
+    return get_chapter_repair_service()
+
+
+# ── 端点 ──
+
+
+@router.get("/novels/{novel_id}/chapter-repair/scan")
+async def scan_short_chapters(
+    novel_id: str,
+    threshold: int = Query(default=4000, ge=100, le=20000, description="字数阈值"),
+    service: ChapterRepairService = Depends(_get_service),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)